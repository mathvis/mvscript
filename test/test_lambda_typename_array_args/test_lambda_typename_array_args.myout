Decl (Variable (VarIdentifier "a") (Just (LambdaT [ArrayT StringT] VoidT)) Nothing)
