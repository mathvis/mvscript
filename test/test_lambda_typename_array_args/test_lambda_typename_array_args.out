Decl (Variable (VarIdentifier "a") (Just lambda[[string]]void) Nothing)
